src/components/EkoSubtitles/EkoSubtitles.jsx

```diff
@@ -31,6 +32,9 @@ export function EkoSubtitles({style}) {
         if (!player) {
             return;
         }
+        if (typeof initialVisibility !== 'undefined') {
```

Comment:
This should happen after the subtitles plugin is inited no?
For the case where the component is inited before the subtitles plugin